index.js

```diff
@@ -18,10 +20,13 @@ module.exports = function (opts) {
 			return;
 		}
 
+		var target = path.relative(opts.abspath, file.path);
```

Comment:
The whole point of `vulcanize` having this option is so that consumer of it won't have to care about this. I think you should rather fix this on `vulcanize`s end.